test_utils.py

```diff
@@ -0,0 +1,141 @@
+import os
+import pytest
+import torch
+from liver_imaging_analysis.engine.utils import VolumeSlicing,Overlay ,calculate_largest_tumor,liver_isolate_crop,find_pix_dim
+import nibabel as nib
+import numpy as np
+
+
+
+# test input files
+volume_dir="test/testdata/volume"
+mask_dir="test/testdata/mask"
+vol_path=os.path.join(volume_dir,'resized_liver.nii')
+mask_path= os.path.join(mask_dir,'resized_mask.nii')   # 0 background, 1 liver, 2 lesion
+mask=nib.load(mask_path)
+volume=nib.load(vol_path)
```

Comment:
It'd be great if you can add these to a fixture as well.

Reply:
I put each one in a separate fixture, is that ok?